test/brsTypes/components/RoAssociativeArray.test.js

```diff
@@ -372,6 +372,20 @@ describe("RoAssociativeArray", () => {
 
                 result2 = aa.get(new BrsString("KeY2"));
                 expect(result2).toBe(v2);
+
+                // check lookupCI method
+                let lookupCI = aa.getMethod("lookupCI");
```

Comment:
can you please add a new test suite for these test cases? Following the same template as we did for other functions, i.e.:
```
describe("lookupCI", () => {
...
}
```
Also it would be interesting to test this function when `SetModeCaseSensitive` is set and when it's not

Reply:
ok, but this method is always is case insensitive, so there is no need test it under different states of `SetModeCaseSensitive`, just need to test if CaseSensitive is enabled
just now tested it and noticed that there is bug
```
brs> aa = {}
brs> aa.SetModeCaseSensitive()
brs> aa["AA"] = 0
brs> ?aa.aa
invalid ' will be 0 in roku
brs> ?aa.lookupCI("aa")
invalid ' will be 0 in roku
brs> ?aa
<Component: roAssociativeArray> =
{
    AA: 0
}

```

Reply:
Thanks for checking, maybe for posterity is worth having its own different test case hoping that one day Roku fixes the bug and we can update the tests appropriately